app/src/components/welcome-page/welcome-page.tsx

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import Button from '../button';
+import Copyright from '../copyright';
+
+export default function WelcomePage(): JSX.Element {
+    return (
+        <div className="ui">
```

Comment:
Использование jsx - не повод отказываться от семантической верстки 🙂
```suggestion
        <main className="ui">
```